api/http/handler/hostmanagement/fdo/handler.go

```diff
@@ -6,26 +6,33 @@ import (
 	"github.com/gorilla/mux"
 
 	httperror "github.com/portainer/libhttp/error"
+	portainer "github.com/portainer/portainer/api"
 	"github.com/portainer/portainer/api/dataservices"
 	"github.com/portainer/portainer/api/http/security"
 )
 
 type Handler struct {
 	*mux.Router
-	DataStore dataservices.DataStore
+	DataStore   dataservices.DataStore
+	FileService portainer.FileService
 }
 
-func NewHandler(bouncer *security.RequestBouncer, dataStore dataservices.DataStore) *Handler {
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
```

Comment:
may I suggest actually leave datastore and adding fileservice? as they're required, it makes it harder to forget to add them

Reply:
agree, I actually changed it to this syntax because most others handlers use it, probably a bigger refactor should be done in all handlers, but I'll change it now for the FDO handler

Reply:
done